### PR TITLE
feat: Loggable.getConnectionInformation() introduced (closes #371)

### LINKED ISSUE #371: Add ConnectionInformation#getConnectionInformation
See also the discussion in #370

### PR DESCRIPTION
**Pull request by typekpb**

### CODE REVIEW

**Comment by typekpb** [comment]
fix for #371

File: src/main/java/com/p6spy/engine/common/Loggable.java [hunk 1]
```diff
@@ -40,5 +40,10 @@
    * @return A connection id which is unique for each {@link java.sql.Connection}
    */
   int getConnectionId();
```
Comment (felixbarny) [inline src/main/java/com/p6spy/engine/common/Loggable.java]: Should we remove this method?

**Comment by typekpb** [comment]
Well, maybe we should, it might be however considered an api breaking
change. But do we care?

File: src/main/java/com/p6spy/engine/common/Loggable.java (hunk 1 above)
Comment (typekpb) [inline src/main/java/com/p6spy/engine/common/Loggable.java]: @felixbarny done

**Pull request merged**